### ISSUE TITLE
Volume Mixer icon isn't changed back after connecting to RDP session

### ISSUE THREAD

**Issue by Gameniti** [1Remote/1Remote#310]
**Describe the bug**
In the Windows Audio Mixer the PRemoteM fader keeps showing the icon of the first connection you have made with rdp instead of changing back to PRemoteM icon/Icon of another RDP session.

**To Reproduce**
1. Completly close PRemoteM (Taskbar icon -> exit)
2. Open PRemoteM
3. Open Windows Audio Mixer
4. Start an RDP Session with a saved icon. You should now see a slider called "\<session name> -  PRemoteM" with the icon of your connected session.
5. Disconnect session. You should now see the PRemoteM Slider called "PRemoteM" still with the icon of your previously connected RDP Session.
6. Connect to another client via RDP (best if it has some other icon). You should now see the name "\<session name> - PRemoteM" but with the icon of your **first** rdp session.
7. Do that as much as you like but the icon of your first rdp session after opening PRemoteM stays until completely exiting the program and start it again.

**Expected behavior**
When changing RDP session the name AND the icon should change correspondingly. 

**Screenshots**
![grafik](https://user-images.githubusercontent.com/26071510/198891959-3e67813e-aa57-4a76-8e22-f908799843c2.png)

**Desktop (please complete the following information):**
 - OS: Windows 10 Pro 21H2
 - PRemoteM Version 0.7.1.6

**Additional context**
I also think it's weird that you only get the volume fader when you start a session and not when opening the program. I mean you still have it if you close every session but it only shows when you start connecting to a client.

**Comment by VShawn** [comment]
Hi thanks for the report, I know this bug for the first time :joy:

**Comment by VShawn** [comment]
failed to find a solution :(

**Comment by itagagaki** [comment]
While trying to find a workaround for this issue, I noticed something odd.

On the "Release / Any CPU" build, if I run it with "Start Without Debugging" and the option "Show current session icon instead of the app icon when connected" is checked, the icon that appears in the Volume Mixer is the icon of that session icon.
However, if  I run it with "Start Debugging" on the same build, the icon in the taskbar is the icon of the session icon, but the icon that appears in the Volume Mixer is the 1Remote app icon.

**Comment by itagagaki** [comment]
Initially there is no 1Remote on the Volume Mixer.
It will appear when we start an RDP session. Good.
But it stays there when the RDP session ends. This is the first thing that annoys me.
I still don't know why.

**Comment by VShawn** [comment]
Yes, when I started investigating, I had no clue and I still haven't been able to find the cause. I didn't even find a Windows API to control the Volume Mixer icon.

**Comment by VShawn** [comment]
> By the way, should the Volume Mixer icon change when you switch the active RDP session? In the first place, should the Volume Mixer display an icon for each RDP session? 

Nope

**Comment by itagagaki** [comment]
The subject says “Taskbar icon”, but it looks like it should be changed to "Volume Mixer icon".

**Comment by itagagaki** [comment]
I noticed that when I launch 1Remote from Explorer and start an RDP session, the volume mixer shows the RDP session icon, but when I launch it from Visual Studio, the volume mixer shows the 1Remote icon. Same exe and same settings. Mysterious.

**Comment by itagagaki** [comment]
As I wrote in the previous comment, when I do "Start Debugging" or "Attach to (1Remote) Process" in Visual Studio, the Volume Mixer shows the 1Remote application icon when the RDP session starts, but when I do "Start Without Debugging" or simply launch 1Remote from another shell, the Volume Mixer shows the RDP session icon.

If the reason for this difference can be determined, I think this issue could be solved. But so far I have no idea.

**Comment by VShawn** [comment]
I edit the code to to avoid copy the voice file to the output folder when build by set dummy.wav as a resource.

![Image](https://github.com/user-attachments/assets/13c3106d-97a1-4f22-a86c-ad1f822977e4)

**Comment by itagagaki** [comment]
> I edit the code to to avoid copy the voice file to the output folder when build by set dummy.wav as a resource.

Hmmm, apparently not quite the same thing.
As for the 1Remote icon being added to the volume mixer by playing dummy.wav when starting 1Remote, both codes give the same result. However, when I launched the Volume Mixer after running 1Remote, it became the Remote icon if it was a real file, but using `GetResourceStream` turned it into an RDP session icon.

**Comment by VShawn** [comment]
![Image](https://github.com/user-attachments/assets/52e2230c-57b7-4234-baa7-5e332f1a2e61)

but it passed my test before I pushed the code. as the screenshoot shows I played some sound on RPD and the icon did not changed.

**Comment by itagagaki** [comment]
Have you run it from Visual Studio? If so, I got the same results. But not when I ran the generated exe.

**Comment by VShawn** [comment]
Based on your suggestion, I downloaded and ran the nightly version, and the icon is still OK.

![Image](https://github.com/user-attachments/assets/51ecc0b4-d370-457f-b883-0baedceee4d3)

**Comment by itagagaki** [comment]
Sorry. I tested again and the result of the icon being a session icon happened in the previous revision as well.

Steps:
1. Make sure the 1Remote and Volume Mixer are not running
2. Run 1Remote
3. Open RDP session
4. Run Volume Mixer

So, I think your modification is OK. My bad.

I thought I had passed the above test as well, but there may have been a mistake. Anyway, I now know that if the Volume Mixer is launched while an RDP session window has the focus, it will become the RDP session icon. As for that, well, it seems a bit impossible to deal with and I don't think it's an issue we should be spending so much effort on, but if I can find a way to deal with it, I'll PR again.

**Comment by VShawn** [comment]
agree, no need to put more effort into this
